paper.tex

```diff
@@ -238,7 +238,7 @@ \section*{Abstract}
 
 Lessons take significant effort to build,
 and even more effort to maintain.
-The collaborative code development methods pioneered by the open source community
+The collaborative code development methods pioneered by the open-source community
```

Comment:
I don't actually have a reference for this, rather I'm trying to make clean that "open-source community" is the concept, as opposed to 'open "source community"' which is a possible interpretation without the hyphenation.